### DISCUSSION TITLE
Please allow uploading bug reports on mobile

### DISCUSSION THREAD

**Discussion by JacksonCampbell99** [community/community#10510] [Mobile]
There is a button to upload a picture on mobile, but all other file types including bug reports are grayed out without the ability to upload them to a GitHub bug report or issue and no additional button for uploading files that aren't pictures. Also, the mobile site doesn't have the upload file banner on the bottom of the text box, but it is there when you request desktop site. It is one of the few things that the desktop site adds, as the two site versions are very similar, and the upload document bar would easily go on the mobile site.
Mobile site without file upload bar:
![Screenshot_20220124-130939_Chrome.jpg](https://user-images.githubusercontent.com/76508020/150865397-371c9b37-f169-4e74-877d-45250622a91c.jpg)
Desktop site on mobile with upload file bar:
![Screenshot_20220124-131015_Chrome.jpg](https://user-images.githubusercontent.com/76508020/150865445-2e2c1e9e-e2af-47ec-b0ef-7c55800fe393.jpg)

Please add an upload option on mobile for files other than pictures and the upload bar on the mobile site.

**Comment by Katsute** [comment]


**Reply by JacksonCampbell99** [reply]
No, I specifically detailed how that button doesn't allow bug file uploads. It is exclusively pictures.

**Reply by Katsute** [reply]
oops didn't notice that

**Comment by JacksonCampbell99** [comment]
Can we get a response?

**Comment by asclepiadae** [comment]
Is there any chance of getting a response to this long-standing issue?? I can see the Images icon in the reply actions bar, and selecting it does allow me to browse Files (strangely called My Documents, which is incorrect for iOS) but when I try to attach a non-image file (say a text crash log) it fails and tells me it cannot attach that file type.

**Comment by iskae** [comment]
Hi @JacksonCampbell99 and everyone 👋 

I am sorry that this request has gone through the cracks and left unanswered 😞 

Unfortunately, this feedback repository is dedicated to GitHub Mobile Native Apps and the screenshots above are related to our Web-App on mobile. With that being said, this feature is actually supported by our GitHub Mobile Native Apps 🎉 You can select an image from your device and share it with the GitHub Mobile app on [iOS](https://apps.apple.com/us/app/github/id1477376905) or on [Android](https://play.google.com/store/apps/details?id=com.github.android&hl=de&gl=US) and you should be able to create an issue with the image that you have just selected 🎉 

Please let me know if I can help any further 🙇 

Cheers!

**Reply by JacksonCampbell99** [reply]
Yes, I submitted feedback for mobile, and you didn't even address it. Read the title. You can't upload bug reports on mobile.

**Reply by JacksonCampbell99** [reply]
You only addressed the screenshot which was to go along with an extra thing I mentioned, separate from the main thing I am requesting. My point was not only can you not upload bug reports on the app, you can't even on the mobile site.

**Reply by iskae** [reply]
Oh, if you are already using the app, we can dig this further, but I am afraid we can't help you with the mobile site. You can reach out to our colleagues in [here](https://github.com/orgs/community/discussions/categories/projects-and-issues?discussions_q=is%3Aunanswered+category%3A%22Projects+and+Issues%22) for requesting features for issues on mobile site 👍 

As mentioned above, unfortunately we are only supporting file selection during issue creation, via sharing the file with GitHub App from another app or via our markdown bar when editing a text content. Can you please check if you can upload the bug report via markdown bar, like in screenshot?

![Screenshot_20230913_103751](https://github.com/community/community/assets/26570736/61698584-0da4-4c6b-982e-8e0468bfcfe1)

For the native apps, can you answer the following questions so we can dig further? Assuming that you are sharing the bug report from another app like a `Files` app to the `GitHub` on mobile:
-  What is the file format of bug reports that you are uploading?
-  How big are the bug reports?
